Toolkit/ArcGISToolkit/AR/ArcGISARView.swift

```diff
@@ -24,7 +24,7 @@ public class ArcGISARView: UIView {
     public let arSCNView = ARSCNView(frame: .zero)
     
     /// The initial transformation used for a table top experience.  Defaults to the Identity Matrix.
-    public private(set) var initialTransformation = AGSTransformationMatrix.identity
+    public var initialTransformation: AGSTransformationMatrix = AGSTransformationMatrix.identity
```

Comment:
This was `private(set) before. I assume you still want that? Also, you're specifying the type twice. While not wrong, it is unnecessary. You could omit one of them. Either:

```suggestion
    public private(set) var initialTransformation = AGSTransformationMatrix.identity
```

or

```suggestion
    public private(set) var initialTransformation: AGSTransformationMatrix = .identity
```